packages/invoices/src/OutboundShipment/DetailView/modals/hooks/useDraftOutboundLines.tsx

```diff
@@ -119,17 +122,22 @@ export const useDraftOutboundLines = (
         })
         .sort(sortByExpiry);
 
-      rows.push(createPlaceholderRow(invoiceId, item?.id));
+      const placeholder = lines.find(
+        ({ type }) => type === InvoiceLineNodeType.UnallocatedStock
+      );
+
+      if (placeholder) {
+        rows.push(
+          createDraftOutboundLine({ invoiceId, invoiceLine: placeholder })
+        );
+      } else {
+        rows.push(createPlaceholderRow(invoiceId, item.id));
+      }
+
```

Comment:
all that really needed to be changed here was: if there was an existing placeholder line, create a standard `draftOutboundLine` seeded with the existing line, otherwise create the placeholder row